src/main/java/org/ibp/api/rest/location/LocationResource.java

```diff
@@ -1,88 +0,0 @@
-
-package org.ibp.api.rest.location;
-
-import java.util.List;
-import java.util.Set;
-
-import org.ibp.api.domain.common.PagedResult;
-import org.ibp.api.domain.location.Location;
-import org.ibp.api.domain.location.LocationType;
-import org.ibp.api.java.location.LocationService;
-import org.ibp.api.rest.common.PaginatedSearch;
-import org.ibp.api.rest.common.SearchSpec;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
-
-import com.wordnik.swagger.annotations.Api;
-import com.wordnik.swagger.annotations.ApiOperation;
-import com.wordnik.swagger.annotations.ApiParam;
-
-@Api(value = "Location Services")
-@RestController
-@RequestMapping("/location")
-public class LocationResource {
-
-	@Autowired
-	private LocationService locationService;
```

Comment:
If org.ibp.api.java.location.LocationService is not used anywhere else then we can also remove it.